src/main/java/picard/sam/HitsForInsert.java

```diff
@@ -27,9 +27,12 @@
 import htsjdk.samtools.SAMTag;
 
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.Comparator;
 import java.util.List;
+import java.util.Objects;
+
+import static java.util.Objects.isNull;
```

Comment:
In general static imports should be avoided as they make code harder to read.

Reply:
Replaced by operators as was discussed [here](https://github.com/broadinstitute/picard/pull/932#discussion_r139697092).